jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -98,11 +97,22 @@ def initiate_dissolution_process(app: Flask):  # pylint: disable=redefined-outer
             app.logger.debug('Skipping job run since there are no businesses eligible for dissolution.')
             return
 
+        # get stage_1 & stage_2 delay configs
+        if (stage_1_delay_config := app.config.get('STAGE_1_DELAY', 0)):
+            stage_1_delay = timedelta(days=int(stage_1_delay_config))
+        else:
+            stage_1_delay = timedelta(days=0)
+
+        if (stage_2_delay_config := app.config.get('STAGE_2_DELAY', 0)):
+            stage_2_delay = timedelta(days=int(stage_2_delay_config))
+        else:
+            stage_2_delay = timedelta(days=0)
```

Comment:
can we update config.py so our delay config entries are like the following?

https://github.com/bcgov/lear/blob/f61c84b50887c3c5e894169743ee0cf3bfdfdde7/queue_services/entity-bn/src/entity_bn/config.py#L78.  I think we can include the default fallback values in code too just in case it doesn't pick it up from 1pass for whatever reason.

Doing this way, we also won't need to provide the fallback outside of config.py.

https://github.com/bcgov/lear/blob/c5d21d3900c1100b35799314514d589d6ef7704c/queue_services/entity-bn/src/entity_bn/bn_processors/change_of_registration.py#L61